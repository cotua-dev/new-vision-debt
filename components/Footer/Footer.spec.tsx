import { render, RenderResult } from '@testing-library/react';
import { Footer } from './index';

describe('Footer Component', (): void => {
    let renderResult: RenderResult;

    beforeAll((): RenderResult => renderResult = render(<Footer/>));

    test('renders', (): void => {
        const { container } = renderResult;
        expect(container.firstChild).toMatchSnapshot();
    });

    afterAll((): void => renderResult.unmount());
});
