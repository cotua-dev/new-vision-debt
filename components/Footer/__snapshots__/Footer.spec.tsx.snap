// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders 1`] = `
<footer
  class="debt-advisors-footer"
>
  <div
    class="footer-content-wrapper"
  >
    <div
      class="contact-us"
    >
      <strong
        class="title"
      >
        Office Hours
      </strong>
      <p
        class="content"
        data-testid="monday-to-friday"
      >
        Monday to Friday 8:00 am to 5:00 pm PST
      </p>
      <p
        class="content"
        data-testid="saturday"
      >
        Saturday Online Only
      </p>
      <p
        class="content"
        data-testid="sunday"
      >
        Sunday Online Only
      </p>
    </div>
    <div
      class="phone-email-submit"
    >
      <strong
        class="title"
      >
        Contact Us
      </strong>
      <div
        class="contact-info"
      >
        <a
          class="info"
          data-testid="phone"
          href="tel:+18009211420"
        >
          800-921-1420
        </a>
      </div>
      <div
        class="flex flex-col"
      >
        <a
          class="duration-200 hover:text-daa-red text-sm"
          data-testid="email"
          href="mailto:info@newvisiondebt.com"
        >
          info@newvisiondebt.com
        </a>
        <a
          class="duration-200 hover:text-daa-red text-sm"
          data-testid="email"
          href="mailto:sales@newvisiondebt.com"
        >
          sales@newvisiondebt.com
        </a>
        <a
          class="duration-200 hover:text-daa-red text-sm"
          data-testid="email"
          href="mailto:support@newvisiondebt.com"
        >
          support@newvisiondebt.com
        </a>
      </div>
    </div>
  </div>
  <div
    class="legal-wrapper"
  >
    <a
      class="legal-link"
      data-testid="terms-of-use"
      href="/terms-of-use"
    >
      Terms of Use
    </a>
    <span>
       / 
    </span>
    <a
      class="legal-link"
      data-testid="privacy-policy"
      href="/privacy-policy"
    >
      Privacy Policy
    </a>
  </div>
</footer>
`;
